client/components/Login.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Form from './Form';
+import { Redirect } from 'react-router';
+
```

Comment:
This login component is very similar to the signup component, with a few slight differences based on whether the user is already in the database. Notice that both components make use of the same `<Form />` component